.github/PULL_REQUEST_TEMPLATE/data_plane_template.md

```diff
@@ -0,0 +1,48 @@
+# Data Plane API - Pull Request
+This PR template is for Data Plane APIs only.
+
+## API Info: The Basics
+Most of the information about your service should be captured in the issue that serves as your [*engagement record*](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/271/Azure-REST-API-Stewardship?anchor=rest-api-stewardship-process).
+
+* Link to engagement record issue: 
+
+Is this review for (select one):
+
+- [ ] a private preview
+- [ ] a public preview
+- [ ] GA release 
+
+### Change Scope
+<sup>This section will help us focus on the specific parts of your API that are new or have been modified. <br/>Please share a link to the design document for the new APIs, a link to the previous Open API document (swagger) if applicable, and the root paths that have been updated. </sup>
+* Design Document: 
+* Previous Open API Doc: 
+* Updated paths:
```

Comment:
Usually a diff works better, but I am not aware what is the best tool. Maybe the commit ref that after which is the outstanding changes (before which would be copy of previous version to current version)?

Reply:
If you mean a diff of the API definition, we should be able to get this either from GitHub directly or from ApiView once the "REST API" view support is implemented.  All we're asking for here is a summary.

Reply:
OK, apiview will definitely help.